### PR TITLE
Initial docusaurus commit

### PR DESCRIPTION
**Pull request by ulrikstrid**
Sync scripts is run like this:
`node .github/workflows/sync-docs.js <repo-name> "<display name>"`

Example usage of sync script:
```
node .github/workflows/sync-docs.js azure-devops-templates "Azure DevOps Templates"
node .github/workflows/sync-docs.js terraform-modules "Terraform Modules"
node .github/workflows/sync-docs.js mqtt-log-stdout "MQTT log stdout"
```

TODO:
- [x] Replace `<br>` with `<br />` in files
- [x] Change assets relative path
- [x] Update links correctly
- [x] Run sync script in GitHub Actions

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,59 +2,16 @@
...
-	cp -r output/azure-devops-templates/assets generated/docs/azure-devops-templates/
+all: install-site build
 
 .SILENT:
 serve: all
-	mkdocs serve -f generated/mkdocs.yml
+	yarn start
 
 .SILENT:
 build: all
```
Comment (phillebaba) [inline Makefile:12]: Should not call all, as it would cause a circular call. If you want to run something before you need to specify them individually.

Comment (ulrikstrid) [reply Makefile]: serve is not in the `all` target so it should be fine right?

Comment (phillebaba) [reply Makefile]: in that case it is fine

**Review by phillebaba — APPROVED** [review]
LGTM

**Pull request merged**